src/measure_theory/measure/measure_space.lean

```diff
@@ -351,6 +351,45 @@ begin
   exact tendsto_at_top_infi (assume n m hnm, measure_mono $ hm hnm),
 end
 
+/-- The measure of the intersection of a decreasing sequence of measurable
+sets indexed by positive reals is the limit of the measures. -/
+lemma tendsto_measure_bInter_pos {s : ℝ → set α}
```

Comment:
The proof works for any dense linear order with `[first_countable_topology]`, not only `ℝ`. Though I'm not sure if we'll ever need it in any other case.